specification/app/data-plane/Microsoft.App.DynamicSessions/preview/2024-10-02-preview/DynamicSessions.json

```diff
@@ -420,6 +420,167 @@
           }
         }
       }
+    },
+    "/listSessions": {
+      "get": {
+        "operationId": "ListSessions_Get",
+        "description": "List the sessions.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "offset",
+            "in": "query",
+            "description": "The offset for pagination purpose.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "limit",
```

Comment:
Same comment for this word "limit" as well. From what I remember there are recommended words to use. Can you check once incase you haven't checked?

Reply:
The above comments are just suggestions, you can ignore them unless API review team asks you to update.

Anyways, according to the guidelines I see that recommended words are "maxpagesize" instead of "limit" and "skip" instead of "offset" words.

Reply:
The guidelines are here: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options

Reply:
updated as per huidelines